client/src/pages/EpisodeDetailsPage.js

```diff
@@ -155,16 +174,18 @@ function EpisodeDetailsPage() {
               " | " +
               episodeDetails.duration_min +
               " Min. | " +
-              episodeDetails.likes +
+              likeCount +
               " Likes"}
           </Stats>
           <ButtonWrapper>
             <AddToSpotifyButton
```

Comment:
In order to faster grasp what element is described, you can name it ``ButtonAddToSpotify``

Reply:
I am thinking about changing the button icon (currently it's a "+" icon) as it only opens a link to Spotify